src/prettytable/prettytable.py

```diff
@@ -2423,9 +2423,9 @@ class TableHandler(HTMLParser):
     def __init__(self, **kwargs) -> None:
         HTMLParser.__init__(self)
         self.kwargs = kwargs
-        self.tables = []
-        self.last_row = []
-        self.rows = []
+        self.tables: list[list] = []
```

Comment:
`list` of `PrettyTable`s?
```suggestion
        self.tables: list[PrettyTable] = []
```